templates/getting_started_typescript/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# First, specify the base Docker image. You can read more about
+# the available images at https://sdk.apify.com/docs/guides/docker-images
+# You can also use any other image from Docker Hub.
+FROM apify/actor-node:16
```

Comment:
the TS dockerfile should use multistage build, compile code and run the `npm start:prod` explicitly, we already have that in the TS template, so lets just use the dockerfile from there

https://github.com/apify/actor-templates/blob/master/templates/example_typescript/Dockerfile

Reply:
Yeah, I know, but for that, the file needs to compile, which it doesn't with the empty result. It should run fine with ts-node, no?